include/opt-sched/Scheduler/sched_basic_data.h

```diff
@@ -83,6 +83,8 @@ enum SPILL_COST_FUNCTION {
   SCF_TARGET
 };
 
+#define MAX_SCF_TYPES 10
+
```

Comment:
there are other macros that are used to define constants. e.g.: MAX_SCHED_PRIRTS .  I agree that using constant variables is more desirable (since they make debugging easier), but I do not want to make the code horrifically inconsistent.